### ISSUE TITLE
LICENCE file gets installed outside of the wheel, easily conflicting with own Licence files

### ISSUE THREAD

**Issue by do3cc** [owais/django-webpack-loader#64]
Steps to reproduce:

```
virtualenv webpack_issue
cd webpack_issue
echo "Free as in beer" > LICENSE
./bin/pip install django-webpack-loader
cat LICENSE
```

the expected output is "Free as in beer" The actual output is the MIT LICENSE

I've looked up the docs for the data_files option and I didn't find a way to define it relative to the wheel location.
Maybe include_package_data would do the trick to add the LICENSE to a better location

**Comment by aleccunningham** [comment]
Fix:
```
$ git clone https://github.com/owais/django-webpack-loader.git
$ cd django-webpaco-loader
$ virtualenv venv
$ source venv/bin/activate
(venv)$ echo "Free as in beer" > LICENSE
```

Make sure to have your virtualenv activated before messing around with anything. Even if I'm misunderstanding your issue, You haven't activated your environment (it could be conflicting with another Python package in site-packages

**Comment by do3cc** [comment]
Hi,
I haven't activated my virtualenv for brevity. This is why I called pip with an absolute path, to ensure to get the right one. This is safe.

Your suggestion is no fix, you did not install the package. Also, run 
```
python setup.y install
```
And my Free as in beer license is gone again.

**Comment by aleccunningham** [comment]
Alright, installed the package via pip inside my virtualenv:


<img width="633" alt="screen shot 2016-07-25 at 8 44 34 pm" src="https://cloud.githubusercontent.com/assets/512428/17122347/a9eebd08-52a8-11e6-94b2-aa9096623eb1.png">

So I'm not sure where the bug is here.

**Comment by do3cc** [comment]
Hi,

the difference between your code example and mine is that you create the Free as in beer LICENSE, after you ran pip install.
So now you overwrote the LICENSE file from django-webpack-loader.
Create the LICENSE file before you run pip install, then you can see the behavior

**Comment by owais** [comment]
<img width="793" alt="screen shot 2016-11-07 at 3 56 34 am" src="https://cloud.githubusercontent.com/assets/46186/20042334/56f646fa-a49e-11e6-8b48-5346a2415f78.png">

@do3cc I couldn't reproduce this. Could you please confirm if this is still an issue? I'm closing the ticket for now.

**Comment by do3cc** [comment]
Checking setup.py, it was fixed in 45793c4d2dcd85adce53940998ea4af033cb7075 by @fladi 
He fixed it in 5 months before I reported the bug, my bugreport coming in just three days before the 0.3.1. release with the fix from @fladi. Why I did not notice that the configuration was gone in master already I don't know.
